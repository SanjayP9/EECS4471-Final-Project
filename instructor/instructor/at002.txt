--Tests winning condition in play mode.
play
pass
pass
pass
pass
pass
pass
pass
pass
pass
pass
pass
pass
pass
pass
pass
pass
move(S)
move(S)
land
