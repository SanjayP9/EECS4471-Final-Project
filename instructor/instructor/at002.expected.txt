  state:0.0, ok
  Welcome! Try test(3,5,7,15,30)
->play
  state:1.0, mode:play, ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MY--   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   P---   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BW--   ----   *---   ----   PW--   
->pass
  state:2.0, mode:play, ok
  Movement:
    [3,P]:[1,5,2]->[5,4,1]
    [8,P]:[4,3,1]->[3,2,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MY--   P---   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BW--   ----   *---   P---   PW--   
->pass
  state:3.0, mode:play, ok
  Movement:
    [5,M]:[3,1,1]->[3,5,1]
    [8,P]:[3,2,1]->[2,1,1]
    [10,B]:[5,1,1]->[5,2,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   PW--   M---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   B---   *---   P---   PW--   
->pass
  state:4.0, mode:play, ok
  Movement:
    [1,P]:[1,3,1]->[2,4,1]
    [2,P]:[1,5,1]->[2,5,2]
    [4,P]:[2,5,1]->[2,4,2]
    [6,P]:[3,4,1]->[2,3,2]
    [7,M]:[4,2,1]->[5,1,1]
    [10,B]:[5,2,1]->[5,1,3]
      reproduced [12,B] at [5,1,4]
      destroyed [7,M] at [5,1,1]
    [11,P]:[5,5,1]->[4,4,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   ----   WP--   PP--   -P--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   -W--   M---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -WBB   ----   *---   P---   -W--   
->pass
  state:5.0, mode:play, ok
  Movement:
    [2,P]:[2,5,2]->[1,5,1]
    [3,P]:[5,4,1]->[1,4,1]
    [6,P]:[2,3,2]->[3,2,1]
    [8,P]:[2,1,1]->[1,5,2]
    [12,B]:[5,1,4]->[5,3,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   ----   P---   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   PP--   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   P---   O---   -W--   M---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -WB-   ----   *B--   ----   -W--   
->pass
  state:6.0, mode:play, ok
  Movement:
    [1,P]:[2,4,1]->[1,3,1]
    [3,P]:[1,4,1]->[1,3,2]
    [5,M]:[3,5,1]->[2,1,1]
      reproduced [13,M] at [2,1,2]
    [6,P]:[3,2,1]->[2,1,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   PP--   ----   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    MMP-   ----   W---   -P--   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -WB-   ----   *B--   ----   -W--   
->pass
  state:7.0, mode:play, ok
  Movement:
    [2,P]:[1,5,1]->[1,1,2]
    [4,P]:[2,4,2]->[2,5,1]
    [8,P]:[1,5,2]->[1,1,3]
    [10,B]:[5,1,3]->[5,1,1]
    [13,M]:[2,1,2]->[1,2,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EPP-   *M--   PP--   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    M-P-   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BW--   ----   *B--   ----   -W--   
->pass
  state:8.0, mode:play, ok
  Movement:
    [2,P]:[1,1,2]->[5,5,1]
    [5,M]:[2,1,1]->[3,1,1]
    [6,P]:[2,1,3]->[2,5,2]
    [12,B]:[5,3,2]->[5,2,1]
      reproduced [14,B] at [5,2,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E-P-   *M--   PP--   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   PP--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MY--   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BW--   BB--   *---   ----   PW--   
->pass
  state:9.0, mode:play, ok
  Movement:
    [1,P]:[1,3,1]->[5,3,2]
    [3,P]:[1,3,2]->[2,4,1]
    [5,M]:[3,1,1]->[4,1,2]
      reproduced [15,M] at [4,1,3]
    [10,B]:[5,1,1]->[2,5,3]
      reproduced [16,B] at [2,5,4]
    [12,B]:[5,2,1]->[1,1,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EBP-   *M--   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   P---   PPBB   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    WMM-   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   -B--   *P--   ----   PW--   
->pass
  state:10.0, mode:play, ok
  Movement:
    [4,P]:[2,5,1]->[2,4,2]
    [6,P]:[2,5,2]->[2,4,3]
    [8,P]:[1,1,3]->[5,5,3]
    [13,M]:[1,2,2]->[2,3,2]
      reproduced [17,M] at [2,3,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EB--   *---   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   WMM-   PPP-   --BB   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    WMM-   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   -B--   *P--   ----   PWP-   
->pass
  state:11.0, mode:play, ok
  Movement:
    [2,P]:[5,5,1]->[4,5,1]
    [4,P]:[2,4,2]->[1,3,1]
    [5,M]:[4,1,2]->[1,4,1]
    [8,P]:[5,5,3]->[5,1,1]
    [12,B]:[1,1,2]->[5,1,3]
    [14,B]:[5,2,2]->[4,1,2]
      destroyed [15,M] at [4,1,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   M---   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   WMM-   P-P-   --BB   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    WB--   -*--   ----   P*P-   P---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *P--   ----   -W--   
->pass
  state:12.0, mode:play, ok
  Movement:
    [3,P]:[2,4,1]->[1,5,1]
    [5,M]:[1,4,1]->[2,5,1]
    [10,B]:[2,5,3]->[1,5,2]
    [13,M]:[2,3,2]->[4,2,1]
    [14,B]:[4,1,2]->[1,1,2]
      reproduced [18,B] at [1,1,3]
    [16,B]:[2,5,4]->[1,1,4]
    [17,M]:[2,3,3]->[4,5,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EBBB   *---   P---   ----   PB--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   --P-   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   ----   P*P-   PM--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *P--   ----   -W--   
->pass
  state:13.0, mode:play, ok
  Movement:
    [2,P]:[4,5,1]->[5,5,1]
    [3,P]:[1,5,1]->[5,1,3]
    [6,P]:[2,4,3]->[3,4,1]
    [10,B]:[1,5,2]->[5,1,4]
    [17,M]:[4,5,2]->[3,1,1]
      reproduced [19,M] at [3,1,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EBBB   *---   P---   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MYM-   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PWP-   ----   *P--   ----   PW--   
->pass
  state:14.0, mode:play, ok
  Movement:
    [2,P]:[5,5,1]->[5,4,1]
    [4,P]:[1,3,1]->[2,2,1]
    [8,P]:[5,1,1]->[1,2,2]
    [13,M]:[4,2,1]->[3,3,2]
    [14,B]:[1,1,2]->[5,5,1]
    [17,M]:[3,1,1]->[4,5,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E-BB   *P--   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   P---   W---   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -YM-   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   M---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -WP-   ----   *P--   P---   BW--   
->pass
  state:15.0, mode:play, ok
  Movement:
    [3,P]:[5,1,3]->[5,5,3]
    [6,P]:[3,4,1]->[3,5,1]
    [14,B]:[5,5,1]->[3,1,1]
      reproduced [20,B] at [3,1,4]
      destroyed [19,M] at [3,1,3]
    [16,B]:[1,1,4]->[1,2,3]
      reproduced [21,B] at [1,2,4]
    [18,B]:[1,1,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E-B-   *PBB   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   P---   W---   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    BY-B   ----   O---   -W--   P---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   M---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   ----   *P--   P---   -WP-   
->pass
  state:16.0, mode:play, ok
  Movement:
    [4,P]:[2,2,1]->[3,2,1]
    [6,P]:[3,5,1]->[3,1,3]
    [14,B]:[3,1,1]->[2,2,1]
    [16,B]:[1,2,3]->[2,3,2]
    [18,B]:[1,1,3]->[5,1,1]
      reproduced [22,B] at [5,1,3]
    [20,B]:[3,1,4]->[2,2,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *P-B   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   BB--   WB--   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -YP-   P---   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   M---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BWB-   ----   *P--   P---   -WP-   
->pass
  state:17.0, mode:play, ok
  Movement:
    [2,P]:[5,4,1]->[4,4,4]
    [3,P]:[5,5,3]->[5,4,1]
    [17,M]:[4,5,1]
      reproduced [23,M] at [4,5,2]
    [18,B]:[5,1,1]->[1,1,2]
    [21,B]:[1,2,4]->[1,1,3]
    [22,B]:[5,1,3]->[2,3,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EBB-   *P--   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   BB--   WBB-   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -YP-   P---   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*PP   MM--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   ----   *P--   P---   -W--   
->move(S)
  state:18.0, mode:play, ok
  Movement:
    [0,E]:[1,1,1]->[2,1,1]
    [4,P]:[3,2,1]->[4,1,2]
    [16,B]:[2,3,2]->[5,1,1]
      reproduced [24,B] at [5,1,3]
    [17,M]:[4,5,1]->[3,1,1]
    [22,B]:[2,3,3]->[4,1,3]
      reproduced [25,B] at [4,1,4]
    [23,M]:[4,5,2]->[5,1,4]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -BB-   *P--   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    E---   BB--   W---   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MYP-   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    WPBB   -*--   ----   P*PP   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BWBM   ----   *P--   P---   -W--   
->move(S)
  state:19.0, mode:play, ok
  Movement:
    [0,E]:[2,1,1]->[3,1,4]
    [3,P]:[5,4,1]->[1,5,1]
    [14,B]:[2,2,1]
      reproduced [26,B] at [2,2,3]
    [20,B]:[2,2,2]->[2,3,2]
      reproduced [27,B] at [2,3,3]
    [24,B]:[5,1,3]->[4,3,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -BB-   *P--   ----   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   B-B-   WBB-   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MYPE   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    WPBB   -*--   B---   P*PP   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BW-M   ----   *P--   ----   -W--   
->land
  state:20.0, mode:play, ok
  Tranquility base here - we've got a life!
